src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -397,21 +402,51 @@
 
 helps['ad group'] = """
 type: group
-short-summary: Manage Azure Active Directory groups.
+short-summary: Manage groups.
 """
 
 helps['ad group create'] = """
 type: command
-short-summary: Create a group in the directory.
+short-summary: Create a group.
 examples:
   - name: Create a group in the directory. (autogenerated)
     text: az ad group create --display-name MyDisplay --mail-nickname MyDisplay
     crafted: true
 """
 
+helps['ad group show'] = """
+type: command
+short-summary: Get the details of a group.
+"""
+
+helps['ad group delete'] = """
+type: command
+short-summary: Delete a group.
+"""
+
+helps['ad group get-member-groups'] = """
+type: command
+short-summary: Get a collection of object IDs of groups of which the specified group is a member.
+"""
```

Comment:
I moved these help messages from `src/azure-cli/azure/cli/command_modules/role/custom.py` to `src/azure-cli/azure/cli/command_modules/role/_help.py` so that they can align with the rest of `az ad` command group. This also allows us to add examples in the future. (docstring in `custom.py` doesn't support examples.)